README.md

```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
 
 `102312307`
 
-Basically places that issue passports, notwithstanding the details (like
-empires which actually issue the passports...)
+Basically places that issue passports, notwithstanding the details (like empires which actually issue the passports...)
```

Comment:
NOTE: In some areas `country` and `dependency` are used interchangeably.

Also... country and county should be a-z sorted.